.ruff.toml

```diff
@@ -1,5 +1,5 @@
 
-#line-length = 90
+line-length = 80
```

Comment:
can we at least compromise on 88 chars, as it is widely used by black? 80 chars are a relic from the 80's a bit of from the current sweetspot.

Reply:
88 is a non-standard line length. Sure, 80 characters is an old requirement, but I find you can pretty much always achieve what you need to and it forces you to keep things clean.

It's personal preference, I suppose, but since I refactored proselint the first time it has used 80 character line lengths.

Reply:
black is the current standard (as it is widely used) in python world - even ruff adopted it and formats the same :)
but ok

Reply:
Black's [completely arbitrary](https://news.ycombinator.com/item?id=17155228), [sarcastically-justified](https://github.com/psf/black/blob/3eea3aad864b83af3b6c477c32f15eb821fe9341/README.md#line-length) 88-character line length is one of its [more controversial "opinions"](https://www.reddit.com/r/Python/comments/9uxo9f/do_you_use_black_with_its_default_line_length_of/), though. 


Particularly as it [violates PEP8](https://github.com/psf/black/issues/498) — an _actual_ standard, not (at best) a "de facto standard" like Black. (Though PEP8 admittedly did itself no favors by specifying _79_ characters instead of 80, something that was met almost immediate, near-universal rejection.)

IIRC line length was also the very first configuration option Black gained, once its original vision of being completely non-configurable collided headfirst with reality.